.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
-  - 5.4
```

Comment:
Guzzle from version 6 requires PHP >= 5.5.0.
Tests were not passing on 5.4, that's why we had to remove it.
Together with Kamil we were unable to install eZ Reco with eZ 4.5.6.